frontend/src/app/api/dashboard.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Dashboard } from '#types/dashboard/dashboard/dashboard';
+import { HttpClient } from '@angular/common/http';
+import { HttpService } from './http.service';
+import { Injectable } from '@angular/core';
+import { NewDashboard } from '#types/dashboard/dashboard/new-dashboard';
+import { Observable } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DashboardService extends HttpService {
+    protected override readonly controllerUri: string = '/dashboards';
+
+    constructor(http: HttpClient) {
+        super(http);
+    }
+
+    public create(newDashboard: NewDashboard): Observable<Dashboard> {
+        return this.post<Dashboard>('', newDashboard);
+    }
+
+    public find(userId: number): Observable<Dashboard[]> {
+        return this.get<Dashboard[]>(`/${userId}`);
```

Comment:
Do you expect that you age going to load dashboards for other users? I guess you may use user id that is stored on backend and do not send user id. Otherwise it would be easier to load dashboards with tasks of other users until you have some additional rules to control whose data user is allowed to load.
It may be actual for other endpoints.